src/Crypto.ts

```diff
@@ -0,0 +1,31 @@
+import * as core from 'webcrypto-core'
+import { askarGetRandomValues } from './askar'
+import { EcdsaCallbackProvider } from './EcdsaCallbackProvider'
+import type { CryptoCallback } from './CryptoCallback'
+
+class Subtle<T> extends core.SubtleCrypto {
+  public constructor(callbacks: CryptoCallback<T>) {
```

Comment:
Why call it callbacks (plural)? Maybe cryptoCallback, or just callback?

Reply:
Ah it was the one place where I did not change it into the singular, good spot